extlib/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ add_subdirectory(fmt)
 add_library(nlohmann_json INTERFACE)
 add_library(nlohmann_json::nlohmann_json ALIAS nlohmann_json)
 target_include_directories(nlohmann_json INTERFACE
-    nlohmann
+    .
```

Comment:
I think this is a bit broader than desired? I'd nest the `nlohmann` directory more.

Reply:
I agree ^^